tests/constructor-bottom/constructor-bottom.helper.js

```diff
@@ -0,0 +1,305 @@
+          }
+        }
+      }
+    `,
+    variables: { constructorElement: addConstructor },
+  });
+  return constructorBottom.data.addConstructorBottom;
+};
+
+const updateConstructorBottom = async (
+  constructorId,
+  operations,
+  newDataConstructorBottom
+) => {
+  const updateConstructor = await operations.mutate({
+    mutation: gql`
+      mutation($id: ID!, $constructorElement: ConstructorBottomInput!) {
+        updateConstructorBottom(
+          id: $id
+          constructorElement: $constructorElement
+        ) {
+          ... on ConstructorBottom {
+            _id
+            name {
+              lang
+              value
+            }
+            material {
+              _id
+            }
+            color {
+              _id
+            }
+            image
+            available
+            default
+          }
+          ... on Error {
+            statusCode
+            message
+          }
+        }
+      }
+    `,
+    variables: {
+      constructorElement: newDataConstructorBottom,
+      id: constructorId,
+    },
+  });
+  return updateConstructor.data.updateConstructorBottom;
+};
+const createConstructorBottomAgain = async (
+  newDataConstructorBottom,
+  operations
+) => {
+  const createConstructorAgain = await operations.mutate({
+    mutation: gql`
+      mutation($constructorElement: ConstructorBottomInput!) {
+        addConstructorBottom(constructorElement: $constructorElement) {
+          ... on ConstructorBottom {
+            _id
+          }
+          ... on Error {
+            statusCode
+            message
+          }
+        }
+      }
+    `,
+    variables: { constructorElement: newDataConstructorBottom },
+  });
+  return createConstructorAgain.data.addConstructorBottom.message;
+};
+
+const updateConstructorB = async (wrongID, operations, addConstructor) => {
+  const updateConstructor = await operations.mutate({
+    mutation: gql`
+      mutation($id: ID!, $constructorElement: ConstructorBottomInput!) {
+        updateConstructorBottom(
+          id: $id
+          constructorElement: $constructorElement
+        ) {
+          ... on ConstructorBottom {
+            _id
+          }
+          ... on Error {
+            statusCode
+            message
+          }
+        }
+      }
+    `,
+    variables: {
+      id: wrongID,
+      constructorElement: addConstructor,
+    },
+  });
+  return updateConstructor.data.updateConstructorBottom.message;
+};
+const deleteConstructorBottom = async (constructorId, operations) => {
+  const deletedConstructor = await operations.mutate({
+    mutation: gql`
+      mutation($id: ID!) {
+        deleteConstructorBottom(id: $id) {
+          ... on ConstructorBottom {
+            _id
+          }
+        }
+      }
+    `,
+    variables: { id: constructorId },
+  });
+  return deletedConstructor.data.deleteConstructorBottom._id;
+};
+const getConstructorBottom = async (newConstructorForQuery, operations) => {
```

Comment:
Can you change `newConstructorForQuery` to id?

Reply:
done